scrapper/src/main.py

```diff
@@ -0,0 +1,54 @@
+from selenium import webdriver
+from selenium.webdriver.chrome.service import Service
+from selenium.webdriver.common.by import By
+from selenium.webdriver.chrome.options import Options
+from selenium.webdriver.support.ui import WebDriverWait
+from selenium.webdriver.support import expected_conditions as EC
+from bs4 import BeautifulSoup
+import json
+
+
+chrome_options = Options()
+chrome_options.add_argument("--headless")
+service = Service('/path/to/chromedriver')
```

Comment:
I don't quite understand why  '/path/to/chromedriver' is provided here, it does not look like a valid path